docs/assistant/security-assistant.asciidoc

```diff
@@ -177,4 +177,17 @@ To enable the knowledge base:
 . Turn on the *Knowledge Base* option.
 . Click *Save*. The knowledge base is now active.
 +
-When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
\ No newline at end of file
+When the knowledge base is active, a quick prompt for {esql} queries becomes available. It provides a good starting point for your {esql} conversations and questions.
+
+[discrete]
+[[ai-assistant-queries]]
+### How to get the most from your queries
```

Comment:
Some suggestions to make the heading slightly more concise:
Optimize your prompts
Optimize AI Assistant responses

Reply:
+1 @natasha-moore-elastic's suggestions, or even just "Get the most from your queries" if you want to keep it conversational; this at least makes it more consistent with Elastic's general active-verb-first format for headings.